docs/sources/reference/components/otelcol.exporter.debug.md

```diff
@@ -0,0 +1,164 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol.exporter.debug/
+description: Learn about otelcol.exporter.debug
+title: otelcol.exporter.debug
+---
+
+<span class="badge docs-labels__stage docs-labels__item">Experimental</span>
+
+# otelcol.exporter.debug
+
+`otelcol.exporter.debug` accepts telemetry data from other `otelcol` components and writes them to the console (stderr).
+You can control the verbosity of the logs.
+
+{{< admonition type="note" >}}
+`otelcol.exporter.debug` is a wrapper over the upstream OpenTelemetry Collector `debug` exporter. 
+If necessary, bug reports or feature requests are redirected to the upstream repository.
+{{< /admonition >}}
+
+Multiple `otelcol.exporter.debug` components can be specified by giving them different labels.
+
+## Usage
+
+```river
+otelcol.exporter.debug "LABEL" { }
+```
+
+## Arguments
+
+`otelcol.exporter.debug` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`verbosity`           | `string` | Verbosity of the generated logs. | `"normal"` | no
+`sampling_initial`    | `int`    | Number of messages initially logged each second. | `2` | no
+`sampling_thereafter` | `int`    | Sampling rate after the initial messages are logged. | `500` | no
+
+The `verbosity` argument must be one of:
+* `"basic"`: A single-line summary of received data is logged to stderr, with a total count of telemetry records for every batch of received logs, metrics or traces.
+* `"normal"`: Currently, this produces the same output as `"basic"` verbosity.
```

Comment:
Why do we have this arg value if it's the same as `basic`? Are there future plans to expand `normal`? Or to have it provide different details at some point? If it will change at some point, this feels like we would introduce a future breaking change for users who use `normal` instead of `basic` and we change the output at some future point.

Or is this for upstream compatibility with the OTel debug exporter?

Reply:
> Why do we have this arg value if it's the same as basic?

It's because upstream has it. It makes using the component simpler for people migrating from/to upstream, and it makes the converters simpler.

> Are there future plans to expand normal? 

I don't know. I guess at some point it'll do something different.

> If it will change at some point, this feels like we would introduce a future breaking change for users who use normal instead of basic and we change the output at some future point.

The component is labeled as "experimental", so breaking changes should be ok. Also, this component is used for debugging and isn't something meant to be run on production on a regular basis, as part of telemetry collection. So breaking changes are not such a big deal.

Reply:
Cool. It was just a moment of "hmmm" as I read this. Matching upstream is enough of a reason.

Reply:
```suggestion
* `"normal"`: Produces the same output as `"basic"` verbosity.
```